src/pages/contributors/yaroslav-afenkin.adoc

```diff
@@ -0,0 +1,96 @@
+I'm more a fan of heavier technical things like Jenkins and other more developer-focused work.
+So, it was easy for me to get excited when I heard they're hiring for a Jenkins project.
+
+== What problems have you worked on in Jenkins
+
+The CSP work was fairly narrow as it was focused on enhancing the Jenkins CSP.
+There are several archetypes of issues and I was familiar with how you approach a certain problem or how you solve it because we had some initiatives when I was originally working in the Jenkins security team.
+We were working on CSP stuff in Jenkins core, so we weren't touching plugins, but most of the issues that I have faced during the CSP project I was already familiar with how to approach them.
+When it comes to the Jenkins security team, the work that I was doing was part of the overall responsibilities of the team such as code review, fixing vulnerabilities in Jenkins core, and security advisories just to name a few.
+
+In regards to the challenges of the CSP project, it was not due to Jenkins or CSP, but JavaScript being Javascript.
+There were some subtle, but very technical JavaScript caveats that needed to be addressed.
+Those problems were partially due to users not willing to leverage what Jenkins already has.
+For instance, it has the Jelly engine for views and people were trying to add JavaScript frameworks, such as React or Angular to that.
+The custom implementations contributed to some issues in their own way because those implementations were not fitting into the archetypes of issues that we were already familiar with and knew how to solve.
+So we had to figure out the framework first and see if there is a mechanism that allows you to enforce CSP.
+Then we had to try using it and depending on how old the technology was, the mechanism could exist or not exist.
+From there, you had to cook up something different to come up with a solution.
+
+==  Is there any part of Jenkins that you've taken a special liking to or found especially interesting?
+
+I think extensibility is the real power of Jenkins.
+If something doesn't exist, you can either look at the plugin ecosystem or in the worst case, you can just write it yourself.
+It's great because you can basically use Jenkins for whatever task you can come up with.
+Taking it one step further, if you want to automate the task you can most likely do it in Jenkins if you spend enough time.
+For instance, performance tests are not a very regular use case as far as I'm concerned, as most people use it to build software or publish, but in my first job we implemented performance tests.
+This is an example of a quirky way to use Jenkins.
+
+While fixing CSP issues, at the same time we would fix vulnerabilities that are out there.
+Security vulnerabilities like Severance or XSS, which have high impact on CVSS scores, so those are the kind of vulnerabilities that need to be addressed.
```

Comment:
Not entirely sure what's meant here, I'll try to rephrase.

It's that some vulnerabilities are caused by incorrect usage of JavaScript (due to developers cutting corners or being unaware), like e.g. usage of `eval` or using inline scripts on HTML elements. These create XSS sinks if unsanitized user input is supplied to them. If CSP is set with a policy that is strict enough then those vulnerabilities wouldn't exist, as scripts won't run.